websites/C/Crowdin/dist/metadata.json

```diff
@@ -15,24 +15,47 @@
 		"en": "Crowdin is a cloud-based solution that streamlines localization management for your team.",
 		"ro": "Crowdin este o soluție bazată pe cloud care eficientizează gestionarea localizării pentru echipa ta.",
 		"nl": "Crowdin is een cloudgebaseerde oplossing die lokalisatiebeheer voor uw team stroomlijnt.",
-		"vi_VN": "Crowdin là giải pháp trực tuyến cho việc quản lý đội ngũ bản địa hoá của bạn."
+		"vi_VN": "Crowdin là giải pháp trực tuyến cho việc quản lý đội ngũ bản địa hoá của bạn.",
+		"pt": "O Crowdin é uma solução de cloud que facilita a gestão de traduções para a tua equipa.",
+		"ru": "Crowdin — это облачное решение, которое упрощает управление локализацией для вашей команды."
 	},
 	"url": [
-		"translate.premid.app",
 		"crowdin.com",
 		"support.crowdin.com",
 		"store.crowdin.com",
 		"status.crowdin.com",
 		"blog.crowdin.com"
```

Comment:
What about links with language codes?
https://ru.crowdin.com/
https://pt.crowdin.com/

Reply:
Could've sworn we had a regex in place, will update